packages/blocks/src/root-block/edgeless/edgeless-root-service.ts

```diff
@@ -594,6 +594,31 @@ export class EdgelessRootService extends RootService {
     return this.surface.getConnectors(id) as ConnectorElementModel[];
   }
 
+  /**
+   * Re-associate bindings for block that have been converted.
+   *
+   * @param oldId - the old block id
+   * @param newId - the new block id
+   */
+  reassociateConnectors(oldId: string, newId: string) {
```

Comment:
@doouding please take a look

Reply:
I think we should try not to add the unnecessary non-common method to the service.  Use the `command` infra or just make it a util function.

Reply:
How to use `command`? Is there an example?

Reply:
You can check at root-service.ts file. `command` is used a lot in the page mode, the edgeless will switch to that infra soon.